docs/sources/installation/helm/install-scalable/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Install the Helm Chart
+menuTitle: Install Loki
+description: Install Loki in scalable mode.
+aliases:
+  - /docs/writers-toolkit/latest/templates/task-template
+weight: 100
+keywords:
+  - keyword
+  - key
+  - word
+---
+
+# Install the Helm Chart
+<!-- vale Grafana.Quotes = NO -->
+<!-- vale Grafana.Quotes = YES -->
```

Comment:
@osg-grafana What is the meaning of these comments?

Reply:
vale is a linter the docs team uses (and is probably run in our CI?>, so this looks to be an override to one of the vale linting rules.